build/ci.mk

```diff
@@ -15,21 +16,21 @@ ci/debug-container: ci/deps
 			-e GPG_MAIL \
 			-e GPG_PASSPHRASE \
 			-e GPG_PRIVATE_KEY_BASE64 \
-			$(BUILDER_TAG) bash
+			$(BUILDER_IMAGE) bash
 
 .PHONY : ci/validate
 ci/validate: ci/deps
 	@docker run --rm -t \
 			-v $(CURDIR):/go/src/github.com/newrelic/nri-$(INTEGRATION) \
 			-w /go/src/github.com/newrelic/nri-$(INTEGRATION) \
-			$(BUILDER_TAG) make validate
+			$(BUILDER_IMAGE) make validate
 
 .PHONY : ci/test
 ci/test: ci/deps
 	@docker run --rm -t \
 			-v $(CURDIR):/go/src/github.com/newrelic/nri-$(INTEGRATION) \
 			-w /go/src/github.com/newrelic/nri-$(INTEGRATION) \
-			$(BUILDER_TAG) make test
+			$(BUILDER_IMAGE) make test
```

Comment:
all these ci/test, ci/validate, ci/snyck, ci/deps, ci/debug-container seems not used anyware. Can we remove them?

Reply:
I don't have enough background to comment for sure that they are not used. I'd assume these are useful for local development. But if your suggestion is to remove them, I can.

Reply:
I suggest to remove them, they create only confusion. you can doublecheck that no one calls those make targets